.github/workflows/smoke-test/smoke-test.sh

```diff
@@ -88,6 +88,17 @@ install_darwin() {
     cp -a "$srcApp" "$(dirname "$destApp")"
     xattr -d -r -s -v com.apple.quarantine "$destApp"
 
+    # Check that the image is compressed
+    local compressionRatio
+    compressionRatio="$(hdiutil imageinfo "$archiveName" \
+        | plutil -convert json -o - - \
+        | jq '.["Size Information"]["Compressed Ratio"]')"
+    if jq --exit-status '. > 0.9' <<< "$compressionRatio"; then
```

Comment:
If we could run `shfmt` against the script, then it would remove a space 😄 
```suggestion
    if jq --exit-status '. > 0.9' <<<"$compressionRatio"; then
```